<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferenceDialog</class>
 <widget class="QDialog" name="PreferenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="preferenceTab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>版本号: 1.0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>账号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="accountLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="unlinkButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>断开连接</string>
     </property>
    </widget>
    <widget class="QLabel" name="spaceLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="usedSpaceLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="maxFileSizeLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLineEdit" name="dirLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>26</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>快盘磁盘路径：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="moveButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>移动</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="runsysche">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>  禁止启动时运行同步</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>作者:ywchen2000           Email: ywchen2000@gmail.com</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Copyright © 2011-2012</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关于IPanForLinux</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>主页:&lt;a href=&quot;http://www.ipanx.net&quot;&gt;http://www.ipanx.net&lt;/a&gt;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
